polyfill/lib/timezone.mjs

```diff
@@ -227,7 +227,7 @@ export class TimeZone {
     return String(GetSlot(this, TIMEZONE_ID));
   }
   toJSON() {
-    return this.toString();
+    return ES.TimeZoneToString(this);
```

Comment:
At first it's a bit confusing that ES.TimeZoneToString isn't used in toString above, but if I think about it, I believe it's correct.

Reply:
It's the other way round (`ES.TimeZoneToString` calls `this.toString` if it exists)